ext/ffi_c/LastError.c

```diff
@@ -49,8 +49,17 @@
 # define USE_PTHREAD_LOCAL
 #endif
 
+#if defined(_WIN32) || defined(__CYGWIN__)
+typedef uint32_t DWORD;
```

Comment:
Why do you declare these functions? This shouldn't be necessary and breaks the MINGW build: https://ci.appveyor.com/project/larskanis/ffi-aofqa/build/1.0.30

Reply:
yeah, I'm stupid, this should obviously be only on cygwin, fixed